### PR TITLE
Performance: More careful polling of graphql data

### PR DESCRIPTION
**Pull request by stackoverfloweth**
PR Checklist:

- [ ] add a short description of what's changed to the top of the `CHANGELOG.md`
- [ ] add/update tests (or don't, for reasons explained below)

### CODE REVIEW

File: src/App.vue
Comment (stackoverfloweth) [inline src/App.vue:102]: note that I've created the infrastructure to make all 6 of these global queries use the new polling state but this PR is only for flows & agents

**Review by zhen0 — DISMISSED** [review]
This is looking good.  One small nitpick from me - when I refresh the page I get the "no flows" banner until all my flows are loaded - this doesn't happen in master/staging.  Can we add a check for loading before we show that banner?  Thanks!

**Review by zhen0 — DISMISSED** [review]
LGTM! Thanks @stackoverfloweth

**Comment by zhen0** [comment]
@cicdw - since this one touches graphQL it needs your approval before we can merge.

**Comment by stackoverfloweth** [comment]
@cicdw let me know if you'd like me to walk through the query changes, there are a lot here that is unrelated to the graphQL change

**Review by cicdw — APPROVED** [review]

**Review by zhen0 — APPROVED** [review]

**Pull request merged**
